### PR TITLE
fix: CHK-2068 modified body for execute call to NPG for retrieve fields

### PR DESCRIPTION
**Pull request by Fernando-Granato**
#### List of Changes
Modified the body of the call to npg for retrieve iframes 
Modified redis properties name to be compliant with version 3 of springboot.
FROM : 
```
spring.redis.host=\${REDIS_HOST}
spring.redis.password=\${REDIS_PASSWORD}
spring.redis.port=\${REDIS_PORT}
spring.redis.ssl=\${REDIS_SSL_ENABLED}
```
TO
```
spring.data.redis.host=\${REDIS_HOST}
spring.data.redis.password=\${REDIS_PASSWORD}
spring.data.redis.port=\${REDIS_PORT}
spring.data.redis.ssl=\${REDIS_SSL_ENABLED}
```

#### Motivation and Context
The change was made to correct information submitted to npg

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
  expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/main/kotlin/it/pagopa/wallet/services/WalletService.kt [hunk 1]
```diff
@@ -78,13 +89,44 @@ class WalletService(
...
+                                    .actionType(ActionType.VERIFY)
+                                    .amount(CREATE_HOSTED_ORDER_REQUEST_VERIFY_AMOUNT)
+                                    .language(CREATE_HOSTED_ORDER_REQUEST_LANGUAGE_ITA)
+                                    .paymentService(
+                                        "CARDS"
+                                    ) // execute call for retrieve by payment method id the service
+                                    // name (GET payment method by id)
+                                    .resultUrl(resultUrl.toString())
+                                    .cancelUrl(cancelUrl.toString())
+                                    .notificationUrl(notificationUrl.toString())
```
Comment (infantesimone) [inline src/main/kotlin/it/pagopa/wallet/services/WalletService.kt:128]: add `captureType` with `IMPLICIT`

**Review by infantesimone — DISMISSED** [review]
just a comment

File: src/main/kotlin/it/pagopa/wallet/services/WalletService.kt [hunk 2]
```diff
@@ -78,13 +89,44 @@ class WalletService(
...
+                                    .customerId(customerId)
+                            )
+                            .paymentSession(
+                                PaymentSession()
+                                    .actionType(ActionType.VERIFY)
+                                    .amount(CREATE_HOSTED_ORDER_REQUEST_VERIFY_AMOUNT)
+                                    .language(CREATE_HOSTED_ORDER_REQUEST_LANGUAGE_ITA)
+                                    .paymentService(
+                                        "CARDS"
+                                    ) // execute call for retrieve by payment method id the service
```
Comment (infantesimone) [inline src/main/kotlin/it/pagopa/wallet/services/WalletService.kt:124]: TODO for another PR?

Comment (Fernando-Granato) [reply src/main/kotlin/it/pagopa/wallet/services/WalletService.kt:124]: Yes, I think it's too onerous to predict it for this sprint.

File: src/main/kotlin/it/pagopa/wallet/services/WalletService.kt (hunk 1 above)
Comment (Fernando-Granato) [reply src/main/kotlin/it/pagopa/wallet/services/WalletService.kt:128]: Yes, I'm sorry, you're right 👌🏻

Comment (Fernando-Granato) [reply src/main/kotlin/it/pagopa/wallet/services/WalletService.kt:129]: Fixed with 3630a2013e4779330a77f89b1ed9205f8c39f2af

File: src/main/kotlin/it/pagopa/wallet/services/WalletService.kt (hunk 2 above)
Comment (infantesimone) [reply src/main/kotlin/it/pagopa/wallet/services/WalletService.kt:125]: yep

**Review by infantesimone — DISMISSED** [review]

**Review by ciuffagianluca — APPROVED** [review]
🚀

File: src/main/kotlin/it/pagopa/wallet/services/WalletService.kt [hunk 3]
```diff
@@ -78,15 +91,55 @@ class WalletService(
...
+                                    .customerId(customerId)
+                            )
+                            .paymentSession(
+                                PaymentSession()
+                                    .actionType(ActionType.VERIFY)
+                                    .amount(CREATE_HOSTED_ORDER_REQUEST_VERIFY_AMOUNT)
+                                    .language(CREATE_HOSTED_ORDER_REQUEST_LANGUAGE_ITA)
+                                    .captureType(CaptureType.IMPLICIT)
+                                    .paymentService(it.first.name)
+                                    .resultUrl(resultUrl.toString())
```
Comment (infantesimone) [inline src/main/kotlin/it/pagopa/wallet/services/WalletService.kt:137]: to add
```
"paymentSession": {
        "actionType": "VERIFY",
        "recurrence": {
            "action": "CONTRACT_CREATION",
            "contractId": "xxx",
            "contractType": "CIT"
        },
```

Comment (Fernando-Granato) [reply src/main/kotlin/it/pagopa/wallet/services/WalletService.kt:144]: Fixed withe 2f4c96554735a7b18cfed61400a6f10f9b09cad

**Review by ciuffagianluca — DISMISSED** [review]
🚀

**Review by ciuffagianluca — APPROVED** [review]

**Pull request merged**